### PR TITLE
Improve examples for DWS-Kueue usage

### PR DESCRIPTION
**Pull request by PBundyra**
Bump Kueue quotas to 1 billion per resource to provide almost unlimited admission

### CODE REVIEW

**Comment by mimowo** [comment]
LGTM from Kueue PoV

**Comment by PBundyra** [comment]
Hi @andrewsykim! Would you be able to approve this one?

File: tutorials-and-examples/workflow-orchestration/dws-examples/dws-queues.yaml [hunk 1]
```diff
@@ -28,20 +28,20 @@ kind: ClusterQueue
...
+        nominalQuota: 1000000000    # "Infinite" quota
       - name: "memory"
-        nominalQuota: 10000Gi # Infinite quota.
+        nominalQuota: 1000000000Gi  # "Infinite" quota
       - name: "nvidia.com/gpu"
-        nominalQuota: 10000  # Infinite quota.
+        nominalQuota: 1000000000    # "Infinite" quota
       - name: "ephemeral-storage"
-        nominalQuota: 10000Ti # Infinite quota.
+        nominalQuota: 1000000000Ti  # "Infinite" quota
```
Comment (andrewsykim) [inline tutorials-and-examples/workflow-orchestration/dws-examples/dws-queues.yaml:44]: Have you manually tested that this works?

Comment (PBundyra) [reply tutorials-and-examples/workflow-orchestration/dws-examples/dws-queues.yaml:44]: Yes

**Pull request merged**